### DISCUSSION TITLE
Custom theme side effect on Vimium

### DISCUSSION THREAD

**Discussion by Dim-Tim-1963** [darkreader/darkreader#9851] [Q&A]
Maybe it's a bug, but I want to ask first. :)  My browser: Waterfox G5 beta2; Lepton theme (default in Waterfox), dark mode.

I tried to switch to custom browser theme (More -> Change the browser theme -> Custom).  I didn't like it, Lepton looks better for me.  I switched back to Default, restarted the browser.  Lepton is back, OK.  But now, when I open Vimium C Vomnibar, it follows the page theme (light or dark) set by Dark Reader, and before it was dark always.  In Vomnibar C settings, I forced dark theme everywhere possible, but without luck...

**Comment by Gusted** [comment]
> Maybe it's a bug, but I want to ask first. :) My browser: Waterfox G5 beta2; Lepton theme (default in Waterfox), dark mode.

Please always open up a bug report either way. Discussions tab are mainly reserved for questions/discussions.

> I tried to switch to custom browser theme (More -> Change the browser theme -> Custom). I didn't like it, Lepton looks better for me. I switched back to Default, restarted the browser. Lepton is back,

> OK. But now, when I open Vimium C Vomnibar, it follows the page theme (light or dark) set by Dark Reader, and before it was dark always. In Vomnibar C settings, I forced dark theme everywhere possible, but without luck...

This sounds like a bug in Vomnibar C settings, if you are forcing the dark theme it shouldn't be light mode. Please open a bug about this in their extension's issue tracker. Feel free to report back if Dark Reader did something unexpected (according to Vomnibar C settings).
